packages/govern-console/src/containers/DAO/components/ModalContents/SignTransfer.tsx

```diff
@@ -0,0 +1,348 @@
+import {
+  GU,
+  LoadingRing,
+  IconConnect,
+  IconCheck,
+  Button,
+  IconRotateLeft,
+  IconExternal,
+} from '@aragon/ui';
+import styled from 'styled-components';
+import { useHistory } from 'react-router';
+import { useFormContext } from 'react-hook-form';
+import { Proposal, ReceiptType } from '@aragon/govern';
+import { useCallback, useEffect, useState, useMemo } from 'react';
+
+import { useWallet } from 'providers/AugmentedWallet';
+import { useTransferContext } from './TransferContext';
+import { proposalDetailsUrl } from 'utils/urls';
+import { CustomTransaction, CustomTransactionStatus } from 'utils/types';
+
+enum SigningState {
+  Processing,
+  Success,
+  Failure,
+}
+
+const SignDeposit: React.FC = () => {
+  const history = useHistory();
+  const { daoIdentifier } = useTransferContext();
+
+  const { getValues } = useFormContext();
+  const { networkName } = useWallet();
+  const { transactions } = useTransferContext();
+  const [txState, setTxState] = useState(SigningState.Processing);
+  const [transactionList, setTransactionList] = useState<CustomTransaction[]>([...transactions]);
+  // const [errorMessage, setErrorMessage] = useState('');
```

Comment:
Remove line

Reply:
if not used for course.... but if the commented line below comes back, this is needed